src/components/CamoComponent.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+	<div class="camo">
```

Comment:
Is this "camo" div really needed?

Reply:
tidied this up, there was some unecessary css as well